changelog.txt

```diff
@@ -1,5 +1,17 @@
 *** Changelog ***
 
+2020.06.30 - version 3.2.0
+* New: Add WooCommerce extensions for installation through the setup wizard [#3297](https://github.com/Automattic/sensei/pull/3297), [#3309](https://github.com/Automattic/sensei/pull/3309), [#3323](https://github.com/Automattic/sensei/pull/3323), [#3322](https://github.com/Automattic/sensei/pull/3322), [#3326](https://github.com/Automattic/sensei/pull/3326), [#3329](https://github.com/Automattic/sensei/pull/3329), [#3332](https://github.com/Automattic/sensei/pull/3332), [#3331](https://github.com/Automattic/sensei/pull/3331), [#3330](https://github.com/Automattic/sensei/pull/3330)
```

Comment:
This should be Sensei LMS extensions, not WooCommerce extension. 😉

Reply:
Updated to the suggestion in the original commit: https://github.com/Automattic/sensei/pull/3335/commits/18656dc957de046c6b8a392a15f3e9268861334c